scripts/docker/buildpack-deps/Dockerfile.ubuntu2004

```diff
@@ -58,5 +59,8 @@ RUN set -ex; \
 
 FROM base
 COPY --from=libraries /usr/lib /usr/lib
+RUN true
```

Comment:
These here might cause slowdown after all... we could work around them differently by having the script build the docker image twice - for me locally a second run of ``docker build`` always works without these hacks, even though the first failed...

Reply:
Yes, let's try to remove them.

Reply:
Looks like it's a known bug: https://github.com/moby/moby/issues/37965.

Reply:
Yeah, it's a known bug. Strangely this seem only be needed for ubuntu2004. But if we remove this, we will not be able to build the images, because of that bug https://github.com/moby/moby/issues/37965

Reply:
Ok, so @aarlt, you'll need to replace one workaround with another :-).
We *should* be able to remove the ``RUN true``'s here, if in the upgrade docker file script we change:
```
docker build "scripts/docker/${IMAGE_NAME}" --file "scripts/docker/${IMAGE_NAME}/Dockerfile.${IMAGE_VARIANT}" --tag "${IMAGE_NAME}"
```
to
```
docker build "scripts/docker/${IMAGE_NAME}" --file "scripts/docker/${IMAGE_NAME}/Dockerfile.${IMAGE_VARIANT}" --tag "${IMAGE_NAME}" || docker build "scripts/docker/${IMAGE_NAME}" --file "scripts/docker/${IMAGE_NAME}/Dockerfile.${IMAGE_VARIANT}" --tag "${IMAGE_NAME}"
```
(or a nicer version of it ;-) - the observation being that locally for me even without these ``RUN true``s the **second** run of docker build always succeeds, while the first one fails...)